sample-app/src/main/java/com/sinch/sample/verification/verifications/Start.java

```diff
@@ -34,10 +35,14 @@ public void run() {
     StartVerificationRequestParameters.Builder<?> builder;
 
     if (method != VerificationMethodType.CALLOUT) {
-      builder = StartVerificationRequestParameters.builder(method);
+      builder = StartVerificationCalloutRequestParameters.builder();
```

Comment:
I'm not sure to understand the logic here: it will always be a `callout` verification either on the line 38, either on line 40. The lines 42 and 44 seem unreachable

Reply:
Yes.
Sample purpose: just change the `method` assignation from line 33 to change the request to be sent to server
Body of function is for educational and highlight how to handle dedicated initialization.

Reply:
Even for educational purpose, I don't think this code will reach its purpose: 
 - when executing the test as is, the method is set to `SMS` (line 33). When reaching the `if` on line 37, the method NOT callout so the builder will become a `CALLOUT` method
 - if the user changes the method to `CALLOUT` on line 33, the `if` clause on line 37 will be false and lead to the line 39 which will resolve to true: the builder will become a `CALLOUT` method but by entering in the `SMS` clause 😵‍💫
 - and when changing to `SEAMLESS` or `FLASH_CALL` it will result in the same output as the first bullet

```suggestion
      if (method == VerificationMethodType.CALLOUT) {
        builder = StartVerificationCalloutRequestParameters.builder();
      } else if (method == VerificationMethodType.SMS) {
        builder = StartVerificationSMSRequestParameters.builder();
      } else if (method == VerificationMethodType.SEAMLESS) {
        builder = StartVerificationSeamlessRequestParameters.builder();
      } else if (method == VerificationMethodType.FLASH_CALL) {
        builder = StartVerificationFlashCallRequestParameters.builder();
      }
```

Reply:
Right, I do not remember where I read/listen this a long time ago: `the "cut and paste" function was the worst invention`
I have to agree 😞

Reply:
Fixed the fix

Reply:
🙌